### PR TITLE
NAS-122432 / 23.10 / add `redis` to `community` train

### PR DESCRIPTION
**Pull request by stavros-k**
- Runs as non-root user but root group

### CODE REVIEW

File: library/ix-dev/community/redis/questions.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            default: false
+        - variable: password
+          label: Password
+          description: Password for Redis.
+          schema:
+            type: string
+            required: true
+            private: true
+            # Character @ is not allowed in password
+            valid_chars: ^((?!@).)*$
```
Comment (Qubad786) [inline library/ix-dev/community/redis/questions.yaml:34]: Should this be in double quotes?

File: library/ix-dev/community/redis/templates/_redis.tpl [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+              enabled: true
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  redis-cli {{ $args }} ping | grep -q PONG
+      initContainers:
+      {{- include "ix.v1.common.app.permissions" (dict "containerName" "01-permissions"
+                                                        "UID" 1001
```
Comment (Qubad786) [inline library/ix-dev/community/redis/templates/_redis.tpl:62]: Fixed uid/gid?

Comment (stavros-k) [reply library/ix-dev/community/redis/templates/_redis.tpl:62]: Yep, container is not playing nice with other uid/his

File: library/ix-dev/community/redis/questions.yaml (hunk 1 above)
Comment (stavros-k) [inline library/ix-dev/community/redis/questions.yaml:34]:
```suggestion
            valid_chars: '^((?!@).)*$'
```

**Review by Qubad786 — APPROVED** [review]

**Pull request merged**